scenarios/mixed_arc_get_agent_activity/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "mixed_arc_get_agent_activity"
+version = "0.1.0"
+edition.workspace = true
+build = "../scenario_build.rs"
+publish = false
+
+[dependencies]
+anyhow = { workspace = true }
+holochain_wind_tunnel_runner = { workspace = true }
+holochain_types = { workspace = true }
+log = { workspace = true }
+kitsune2_api = { workspace = true }
+kitsune2_core = { workspace = true }
```

Comment:
Only `anyhow`, `holochain_wind_tunnel_runner`, and `holochain_types` are used, right?